src/components/application_manager/rpc_plugins/rc_rpc_plugin/test/commands/set_interior_vehicle_data_request_test.cc

```diff
@@ -231,4 +232,89 @@ TEST_F(
+       Execute_ValidWithSettableParams_SUCCESSSendToHMI) {
+  MessageSharedPtr mobile_message = CreateBasicMessage();
+  auto& msg_params =
+      (*mobile_message)[application_manager::strings::msg_params];
+  msg_params[message_params::kModuleData][message_params::kModuleType] =
+      mobile_apis::ModuleType::CLIMATE;
+
+  msg_params[message_params::kModuleData][message_params::kClimateControlData] =
+      smart_objects::SmartObject(smart_objects::SmartType_Map);
+
+  msg_params[message_params::kModuleData][message_params::kClimateControlData]
+            [message_params::kClimateEnable] = true;
+
+  // Expectations
+  EXPECT_CALL(mock_policy_handler_, CheckModule(kPolicyAppId, _))
+      .WillOnce(Return(rc_rpc_plugin::TypeAccess::kAllowed));
+
+  EXPECT_CALL(mock_hmi_capabilities_, rc_capability())
+      .WillOnce(Return(nullptr));
+
+  auto message_from_mobile = CreateBasicMessage();
+
+  EXPECT_CALL(mock_rpc_service_, ManageHMICommand(_))
+      .WillOnce(DoAll(SaveArg<0>(&message_from_mobile), Return(true)));
+
+  std::shared_ptr<
+      rc_rpc_plugin::commands::SetInteriorVehicleDataRequest> command =
+      CreateRCCommand<rc_rpc_plugin::commands::SetInteriorVehicleDataRequest>(
+          mobile_message);
+  command->Run();
+
+  auto& msg_params_from_mobile =
+      (*message_from_mobile)[application_manager::strings::msg_params];
+
+  const bool climate_enable =
+      msg_params_from_mobile[message_params::kModuleData]
+                            [message_params::kClimateControlData]
+                            [message_params::kClimateEnable].asBool();
+  EXPECT_TRUE(climate_enable);
+}
+
+TEST_F(SetInteriorVehicleDataRequestTest,
+       Execute_ValidWithSettableParams_SUCCESSSendToHMI_HDChannel) {
+  MessageSharedPtr mobile_message = CreateBasicMessage();
+  auto& msg_params =
+      (*mobile_message)[application_manager::strings::msg_params];
+  msg_params[message_params::kModuleData][message_params::kModuleType] =
+      mobile_apis::ModuleType::RADIO;
+
+  msg_params[message_params::kModuleData][message_params::kRadioControlData] =
+      smart_objects::SmartObject(smart_objects::SmartType_Map);
+
+  const std::uint32_t hd_Channel = 2u;
+  msg_params[message_params::kModuleData][message_params::kRadioControlData]
+            [message_params::kHdChannel] = hd_Channel;
+
+  // Expectations
+  EXPECT_CALL(mock_policy_handler_, CheckModule(kPolicyAppId, _))
+      .WillOnce(Return(rc_rpc_plugin::TypeAccess::kAllowed));
+
+  EXPECT_CALL(mock_hmi_capabilities_, rc_capability())
+      .WillOnce(Return(nullptr));
+
+  auto message_from_mobile = CreateBasicMessage();
+
+  EXPECT_CALL(mock_rpc_service_, ManageHMICommand(_))
+      .WillOnce(DoAll(SaveArg<0>(&message_from_mobile), Return(true)));
+
+  std::shared_ptr<
+      rc_rpc_plugin::commands::SetInteriorVehicleDataRequest> command =
+      CreateRCCommand<rc_rpc_plugin::commands::SetInteriorVehicleDataRequest>(
+          mobile_message);
+  command->Run();
```

Comment:
@IGapchuk Check all tests for call Init() before Run()

Reply:
@ZhdanovP @AByzhynar Fixed in a57b72856e60e30b65c6435d169cee8d10729660.